apps/hash-ai-worker-ts/src/activities/infer-entities/infer-entities-system-message.ts

```diff
@@ -14,13 +10,29 @@ export const inferEntitiesSystemMessage: OpenAI.ChatCompletionSystemMessageParam
       and do not rely on other information about the entities in question you may know. 
     The entities you create must be suitable for the schema chosen 
       – ignore any entities in the provided text which do not have an appropriate schema to use. 
-      The keys of the entities 'properties' objects are URLs which end in a trailing slash. This is intentional –
-      please do not omit the trailing slash.
+
     The user may respond advising you that some proposed entities already exist, and give you a new string identifier for them,
       as well as their existing properties. You can then call update_entities instead to update the relevant entities, 
       making sure that you retain any useful information in the existing properties, augmenting it with what you have inferred. 
+    
     The more entities you infer, the happier the user will be!
     Make sure you attempt to create entities of all the provided types, if there is data to do so!
     The user has requested that you fill out as many properties as possible, so please do so. Do not optimise for short responses.
-  `),
-  };
+    
+    Be extremely careful when specifying units for data, that the specified unit is correct.
+    Locations where the units might be specified:
+      - alongside the value
+      - in a column header of a table
+      - in a another cell on the same table row
+      - elsewhere in the content.
+
+    Be careful, you may need to combine units from multiple sources to infer the correct unit for a value.
+
+    Do not infer units from the name of the properties, or what you believe they should be.
+    You must base units on the information provided in the text.
+
+    If a property value is numeric, and some multiplier is specified (e.g. "millions (m)", "thousands (k)"), you must
+    either convert the value to the base unit.
```

Comment:
Sorry I previously had an `or` here which I removed. I've removed this sentence from the prompt in https://github.com/hashintel/hash/pull/4330/commits/a505349f25a1d0a7eedf8316f2d8e66b85fa1ea5 because conversion is not actually something I think is a good idea to encourage here, with the absence of a separate step for identifying the units correctly.